test/neon_animated_pages_test.dart

```diff
@@ -0,0 +1,45 @@
+@TestOn('browser')
+library polymer_elements.neon_animated_pages_test;
+
+import 'package:polymer_interop/polymer_interop.dart';
+import 'package:polymer_elements/neon_animated_pages.dart';
+import 'package:test/test.dart';
+import 'package:web_components/web_components.dart';
+import 'common.dart';
+import 'dart:html';
+
+main() async {
+
+  await initWebComponents();
+
+  group('<neon-animated-pages>', () {
+
+    group('basic',(){});
```

Comment:
lets just remove this empty group even though it does exist in the original test.